pages/checkout.php

```diff
@@ -498,7 +498,7 @@
 	<?php
 		global $recaptcha, $recaptcha_publickey;
 		if ( $recaptcha == 2 || $recaptcha == 1 ) {
-			echo pmpro_recaptcha_get_html($recaptcha_publickey, NULL, true);
+			pmpro_recaptcha_get_html($recaptcha_publickey);
```

Comment:
We need to be careful with these kinds of changes because of sites that may be using a custom checkout page template. Think that this particular change *should* be ok if the new `pmpro_recaptcha_get_html()` function already echos the output and the function overload should be ok.